pkg/ddevapp/config_test.go

```diff
@@ -534,14 +534,14 @@ func TestConfigValidate(t *testing.T) {
 	app.DdevVersionConstraint = ">= 1.twentythree"
 	err = app.ValidateConfig()
 	assert.Error(err)
-	assert.Contains(err.Error(), "constraint that is not valid")
+	assert.Contains(err.Error(), "constraint is not valid")
 	app.DdevVersionConstraint = ""
 
 	versionconstants.DdevVersion = "v1.22.0"
 	app.DdevVersionConstraint = ">= 1.23"
 	err = app.ValidateConfig()
 	assert.Error(err)
-	assert.Contains(err.Error(), "project has a DDEV version constraint of '>= 1.23' and the version of DDEV you are using ('v1.22.0') does not meet the constraint")
+	assert.Contains(err.Error(), "your DDEV version 'v1.22.0' doesn't meet the constraint '>= 1.23'")
```

Comment:
Interestingly, there has been long-running problems with version comparisons that don't have patch version. I guess gien this test maybe that's resolved. So I would have always used `v1.23.0`.